src/index.ts

```diff
@@ -8,6 +8,9 @@ import { checkForUrlWithCode } from "./zentrum";
 import { tmpdir } from "os";
 import * as path from "path";
 const debug = Debug("impftermin:main");
+import { bgRedBright, whiteBright } from "chalk";
+
+export const error = (...text: unknown[]) => bgRedBright(whiteBright(...text));
```

Comment:
Could you give this a more descriptive name like `colorErrorText` or something?

Reply:
How about `coloredError`?